blog/_posts/2020-11-24-ranluxpp.md

```diff
@@ -0,0 +1,146 @@
+---
+title: "Implementing and tuning RANLUX++ for ROOT"
+layout: archive
+author: Jonas Hahnfeld
+---
+
+ROOT includes different implementations of pseudorandom number generators (PRNGs):
+ * [`TRandom`][TRandom] is a Linear Congruential Generator (LCG) based on the BSD `rand` generator.
+ * [`TRandom1`][TRandom1] implements the RANLUX generator described by Fred James in 1990.
+ * [`TRandom3`][TRandom3] is based on the Mersenne Twister generator.
+   By default, this implementation is used for `gRandom`.
+ * `TRandomMixMax` is an instantiation of the parametric [`MixMaxEngine`][MixMaxEngine].
+
+[TRandom]: https://root.cern/doc/master/classTRandom.html
+[TRandom1]: https://root.cern/doc/master/classTRandom1.html
+[Trandom3]: https://root.cern/doc/master/classTRandom3.html
+[MixMaxEngine]: https://root.cern/doc/master/classROOT_1_1Math_1_1MixMaxEngine.html
+
+This post discusses the implementation of the recently added `TRandomRanluxpp`.
```

Comment:
In order to have a nicer "preview" on the front page (see https://root.cern/465 ) I would suggest to expand this last sentence a bit and put it first (for example, just to give you an idea: "ROOT offers many different random number generators, and we just added one that we think you might be interested in! In this post I will go over what RANLUX++ is and how I recently implemented it and optimized it for ROOT.").

The list of available PRNGs could then be moved under `## Why so many PRNGs?`, or just under this first introductory paragraph.

Reply:
Adapted, let me know what you think!

Reply:
perfect!